zellij-utils/src/data.rs

```diff
@@ -237,19 +293,36 @@ pub struct Style {
     pub rounded_corners: bool,
 }
 
+// FIXME: Poor devs hashtable since HashTable can't derive `Default`...
```

Comment:
How should we derive a new trait for a type from the `std` library? Since we own neither the `Default` trait, not the `HashMap`, the orphan rule keeps us from doing that. Or am I missing something?